src/python/grpcio/grpc/_server.py

```diff
@@ -447,6 +475,13 @@ def _serialize_response(rpc_event, state, response, response_serializer):
         return serialized_response
 
 
+def _state_to_send_message_op_flags(state):
+    if state.disable_next_compression:
```

Comment:
It seems once this value is set to `True`, there is no way to get it back to `False`.

Reply:
Very good catch. Fixed.